chirho/counterfactual/ops.py

```diff
@@ -23,10 +25,61 @@ def split(obs: T, acts: Tuple[Intervention[T], ...], **kwargs) -> T:
     return scatter(act_values, event_dim=kwargs.get("event_dim", 0))
 
 
+@pyro.poutine.runtime.effectful(type="undo_split")
+def undo_split(
+    antecedents: Iterable[str] = None, event_dim: int = 0
+) -> Callable[[T], T]:
+    """
+    A helper function that undoes an upstream `chirho.counterfactual.ops.split`
```

Comment:
Explain that `undo_split` is meant to be used to create arguments to pass to `intervene`/`split`/`preempt`

Reply:
done